invoices/models.py

```diff
@@ -241,7 +216,7 @@ def date(self):
 
     @property
     def full_name(self):
```

Comment:
Should full name include week and year as well, or is skipping those intentional?

Reply:
Yes it is intentional, weekly_report_page.html displays both full_name and date (incl. year & week). This is similar to what is done in invoice_page.html